src/utils/config.rs

```diff
@@ -207,25 +212,55 @@ impl GlobalConfig {
             "Circular"
         };
         let edit_style = if self.edit_style { "Emacs" } else { "Vi" };
-        let index_state = self.index_state.read().to_string();
         let version = crate::get_version();
         let version_long = version.long();
         let network_string = self
             .network()
             .map(|value| format!("{:?}", value))
             .unwrap_or_else(|| "unknown".to_string());
-        let url_string = format!("{} (network: {})", self.get_url(), network_string);
+        let ckb_tip = if fast_mode {
+            "loading...".to_string()
+        } else {
+            match CkbRpcClient::new(self.get_url()).get_tip_block_number() {
+                Ok(number) => format!("#{}", number.value()),
+                Err(_err) => "#unknown".to_string(),
```

Comment:
suggest to display the detail error, same as indexer rpc error handling:
```suggestion
                Err(err) => err.to_string(),
```